meta/src/main/java/org/apache/rocketmq/mqtt/meta/raft/processor/CounterStateProcessor.java

```diff
@@ -34,9 +34,9 @@
 public class CounterStateProcessor extends StateProcessor {
```

Comment:
What does this class do

Reply:
This is a demo that state machine how to work